solutions/kubernetes/cortx-deploy-functions.sh

```diff
@@ -242,29 +235,7 @@ function execute_prereq(){
     popd    
 }
 
-function usage(){
-    cat << HEREDOC
-Usage : $0 [--setup-worker, --setup-primary, --third-party, --cortx-cluster, --destroy, --status]
-where,
-    --setup-worker - Setup k8 worker node for CORTX deployment
-    --setup-primary - Setup k8 primary node for CORTX deployment
-    --third-party - Deploy third-party components
-    --cortx-cluster - Deploy Third-Party and CORTX components
-    --destroy - Destroy CORTX Cluster
-    --status - Print CLUSTER status
-HEREDOC
-}
-
-
-ACTION="$1"
-if [ -z "$ACTION" ]; then
-    echo "ERROR : No option provided"
-    usage
-    exit 1
-fi
-
-function setup_primary_node(){
-echo "---------------------------------------[ Setting up Primary Node $HOSTNAME ]--------------------------------------"
```

Comment:
Don't we need this print statement?

Reply:
Removed that because when cortx-deploy.sh executes --setup-primay function it already prints following:
![image](https://user-images.githubusercontent.com/91177647/160072798-7ffefae4-d891-4c00-949b-9bfcc0f63c18.png)

printing in cortx-deploy-functions.sh will deupicate the same message right below it.